packages/react-urql/examples/1-getting-started/src/index.tsx

```diff
@@ -2,11 +2,13 @@ import React, { FC, StrictMode } from 'react';
 import * as ReactDOM from 'react-dom';
 import { createClient, Provider, defaultExchanges } from 'urql';
 import { devtoolsExchange } from '@urql/devtools';
-import { Home } from './Home';
+import { Home } from './pages';
 import './index.css';
 
+navigator.serviceWorker.register('./service-worker.ts', { scope: '/sw' });
```

Comment:
The browser will likely not update properly unless you have `.then(reg => reg.update())` here... at least that's how I remember it?

Reply:
Nope, this is working fine